src/pages/CatInfoPage.spec.js

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import CatInfoPage from './CatInfoPage.js';
+
+describe('CatInfoPage', () => {
+  it('renders a headline and a return button with an image', () => {
+    render(
+      <MemoryRouter>
+        <CatInfoPage />
+      </MemoryRouter>
+    );
+
+    const backButton = screen.getByRole('button');
+    const headlineCatBreeds = screen.getByText(/cat breeds/i);
+    const imgInButton = screen.getByAltText(
+      'cat icon with the head to the left'
+    );
+
+    expect(backButton).toBeInTheDocument();
+    expect(headlineCatBreeds).toBeInTheDocument();
+    expect(imgInButton).toBeInTheDocument();
+  });
+
+  it('renders an unordered list', () => {
```

Comment:
you could test if the single items are also displayed as expected 😊

Reply:
thanks for that advice, i would like to, so I will soon ask for your help for this 😊

Reply:
oh oh 😂🙈